File: 백준/Bronze/17210. 문문문/문문문.c
#include <stdio.h>

typedef long long ll;

ll n;
int way;

int main() {
    scanf("%lld %d", &n, &way);

    if(n > 5) {
        printf("Love is open door");
    }
    else {
        for(int i = 1; i < n; i++){
            way = 1 - way;
            printf("%d\n", way);
        }
    }
    return 0;
}

File: 백준/Bronze/17210. 문문문/문문문.py
num_door = int(input())
cur_door = int(input())

if num_door >= 6:
    print("Love is open door")
else:
    for _ in range(num_door-1):
        if cur_door == 0:
            print("1")
            cur_door = 1
        else:
            print("0")
            cur_door = 0